Bottom: 2c3736ca40e5712601c20aeb6e698a397aa2b8ef
Top:    f17e1ab02156d687f8752883ecdc495f911c72ea
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-13 00:30:21 +0100

Refresh of speedup_poly_poly_checking

---

diff --git a/src/find.c b/src/find.c
index e8d08c5..3fcfe9d 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2499,6 +2499,8 @@ LookupLOConnectionsToPolygon (PolygonTypePtr Polygon, Cardinal LayerGroup)
       /* handle normal layers */
       if (layer < max_layer)
         {
+          info.layer = layer;
+
           /* check all polygons */
           if (setjmp (info.env) == 0)
             r_search (LAYER_PTR (layer)->pour_tree,
@@ -2507,7 +2509,6 @@ LookupLOConnectionsToPolygon (PolygonTypePtr Polygon, Cardinal LayerGroup)
           else
             return True;
 
-          info.layer = layer;
           /* check all lines */
           if (setjmp (info.env) == 0)
             r_search (LAYER_PTR (layer)->line_tree,
